hack/make/.build-deb/docker-engine.install

```diff
@@ -9,3 +9,4 @@ contrib/init/systemd/docker.socket lib/systemd/system/
 contrib/mk* usr/share/docker-engine/contrib/
 contrib/nuke-graph-directory.sh usr/share/docker-engine/contrib/
 contrib/syntax/nano/Dockerfile.nanorc usr/share/nano/
+contrib/apparmor/* etc/apparmor.d/
```

Comment:
@jfrazelle you were planning to take over adding the `dh-apparmor` bits to actually plumb this into the maintscripts automatically post-merge, right?